### PR TITLE
sketchy validator patch

### PR DESCRIPTION
**Pull request by bryanchriswhite**

### CODE REVIEW

**Comment by braydonf** [comment]
What up with this? There are some known issues with the regex https://github.com/Storj/service-storage-models/issues/75 but what issue is this solving?

File: lib/models/storage-event.js [hunk 1]
```diff
@@ -22,11 +22,7 @@ var StorageEvent = new mongoose.Schema({
   user: {
     type: String, 
     ref: 'User',
-    required: true,
```
Comment (braydonf) [inline lib/models/storage-event.js:25]: This should remain as true. And email validation should be consistent across all models. The validation should be fixed not removed.

**Review by braydonf — CHANGES_REQUESTED** [review]

File: lib/models/storage-event.js (hunk 1 above)
Comment (bryanchriswhite) [reply lib/models/storage-event.js:25]: @braydonf it is still true.. see line 25

**Comment by braydonf** [comment]
We may want to relax the regex to only care about the maximum length of the characters (<64 characters>@<255 characters>), as mentioned in https://github.com/Storj/service-storage-models/pull/45. Validation will ultimately be tested by if the address can receive email.

File: lib/models/storage-event.js (hunk 1 above)
Comment (braydonf) [reply lib/models/storage-event.js:25]: Oh... okay. But why remove the validation?

**Comment by phutchins** [comment]
This is a temporary fix... Merging.

**Pull request merged**